### DISCUSSION TITLE
Running LDT and LIS for multiple domains

### DISCUSSION THREAD

**Discussion by GGDLW** [NASA-LIS/LISF#1384] [General]
Hello,

Is it possible to run LDT and LIS for multiple domains? If so, how would I format the input file? Would I do it like I would do a namelist file from WPS or WRF or do I need to do something a little different.

**Comment by dmocko** [comment]
Hi @GGDLW 

Yes, you can run LDT/LIS for multiple domains (aka, nests).

See the LDT Users' Guide and look for the config entry `LIS number of nests:`.
    https://nasa-lis.github.io/LISF/LDT_users_guide/LDT_users_guide.html

You could set this to "2" to have two nests.  Then, you would repeat config entries, one value for the first (or outer) nest, and then a second value (with white space as a delimiter) for the second (or inner) nest.  Here is a partial example:

```
LDT running mode:                       "LSM parameter processing"
Processed LSM parameter filename:       ./lis_input.d01.nc   ./lis_input.d02.nc
LIS number of nests:                    2
Surface model types:                    "LSM"  "LSM"
Land surface model:                     "Noah-MP.4.0.1"  "Noah-MP.4.0.1"
Map projection of the LIS domain:       latlon  latlon
LL run domain lower left lat:             30.0   40.0
```

Once LDT completes, you will see two different lis_input files, one for domain 1 (`.d01.nc`) and one for domain 2 (`.d02.nc`).

In LIS, you then repeat this process, specifying the number of nests, and then repeating config entries as needed.

Please try this out, and let us know if you see any issues.

Post your ldt.config and/or lis.config and any log messages if you encounter an issue.

**Comment by GGDLW** [comment]
Thank you for the help,

I am currently trying to run for nested domain. I am curious if I have to repeat all entries for a nested run. Here is my config file and my log file. 
[ldt.config.prelis.txt](https://github.com/NASA-LIS/LISF/files/12221561/ldt.config.prelis.txt)
[ldtlog_prelis1.0000.txt](https://github.com/NASA-LIS/LISF/files/12221563/ldtlog_prelis1.0000.txt)
